.github/workflows/deploy-gh-pages.yml

```
@@ -0,0 +1,27 @@
+name: Deploy to GithubPages
+
+# on:
+#   push:
+#     branches:
+#       - master
+on: pull_request
+permissions:
+    contents: write
+jobs:
+    deploy-to-gh-pages:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
```

не очень важно, но лучше держать версии актуальные
```suggestion
              uses: actions/checkout@v4
```